content/en/docs/concepts/services-networking/service.md

```diff
@@ -158,9 +158,9 @@ Each port definition can have the same `protocol`, or a different one.
 
 ### Services without selectors
 
-Services most commonly abstract access to Kubernetes Pods, but they can also
-abstract other kinds of backends.
-For example:
+Services most commonly abstract access to Kubernetes Pods thanks to the selector,
+but when used without a selector, the Service can abstract other kinds of backends, 
```

Comment:
Should this paragraph also state that without a selector and a corresponding Endpoints object that a Service can abstract other kinds of backends.
This is stated further down the page so I'm ok with or without it
```suggestion
but when used with a corresponding Endpoints object and without a selector, the Service can abstract other kinds of backends, 
```

Reply:
Thank you. It's clearer to reference the Endpoints and helps the reader figure out what they need to do. I'll push an update shortly.